src/views/open-api-docs-view-v2/components/operation-content/style.module.css

```diff
@@ -1,3 +1,12 @@
+/* HEADER AREA */
+
+.header {
```

Comment:
`OperationContent` header area styles needed some minor updates to accommodate the new `versionSwitcherSlot`.